packages/babel-runtime/scripts/build-dist.js

```diff
@@ -107,7 +107,8 @@ function buildHelper(helperName, modules, useBuiltIns) {
       : null;
   const sourceType = modules === "commonjs" ? "script" : "module";
 
-  const tree = t.program(helpers.get(helperName, id).nodes, [], sourceType);
+  const helper = helpers.get(helperName, null, id);
+  const tree = t.program(helper.nodes, [], sourceType);
```

Comment:
This should have a getDependency callback to insert `require` calls for the imported helper, for the ES5 helper files.

Reply:
Oops